config/common/server.yaml

```diff
@@ -0,0 +1,267 @@
+{{ range $index, $Map := .ServerCount }}
+persistentVolumeClaims:
```

Comment:
not sure if this can (or should) be included as a "common" kieserver component. Only prod uses postgres. Authoring, for example, uses a singular mysql DB.